language_load.py

```diff
@@ -38,49 +44,56 @@ def iterencode(self, o, _one_shot=False):
             yield s
 
 for filename in glob.iglob(args.objects + '/**/*.json', recursive=True):
-    file = open(filename)
+    file = open(filename, encoding="utf8")
     data = json.load(file)
     file.close()
 
     if not 'strings' in data:
-        print("No strings in " + data['id'] + " -- skipping")
+        if verbose:
+            print("No strings in " + data['id'] + " -- skipping")
         continue
 
     if not data['id'] in strings_by_object:
-        print("No translations for " + data['id'] + " in dump file -- skipping")
+        if verbose:
+            print("No translations for " + data['id'] + " in dump file -- skipping")
         continue
 
     updated = False
     for string_key in data['strings']:
 
         if not string_key in strings_by_object[data['id']]:
-            print("No translation for " + data['id'] + " string '" + string_key + "' in dump file -- skipping")
+            if verbose:
+                print("No translation for " + data['id'] + " string '" + string_key + "' in dump file -- skipping")
             continue
 
         if not fallback_language in data['strings'][string_key]:
-            print("No en-GB reference for " + data['id'] + " string '" + string_key + "' in dump file -- probably shouldn't exist; skipping")
+            if verbose:
+                print("No en-GB reference for " + data['id'] + " string '" + string_key + "' in dump file -- probably shouldn't exist; skipping")
             continue
 
         if not language_to_import in data['strings'][string_key]:
             if strings_by_object[data['id']][string_key] == data['strings'][string_key][fallback_language]:
                 # TODO: Is this desirable behaviour?
```

Comment:
I don't think it is, because leaving them out will lead to them being marked as incomplete by the dump script.

Reply:
Hmm, I think I added this to prevent importing untranslated strings, e.g. when a language pack is at a stage where 90% of strings are untranslated. I agree it will make it harder to do stats, though. Something for another PR, then.

Reply:
I think it is problematic because some things have the same translation for some languages and not for others and if we go towards having different files for each language on Localisation, people might want to have it "repeated" just to acknowledge that it was taken care of.